python/tests/compat/test_mantid.py

```diff
@@ -151,6 +149,27 @@ def test_Workspace2D_not_common_bins_masks(self):
             ds.masks["spectrum"].values[0:3],
             [True, True, True])
 
+    def test_Workspace2D_with_separate_monitors(self):
+        filename = MantidDataHelper.find_file("WISH00016748.raw")
```

Comment:
Minor point, but given loading times, could we cache-reuse the same input?

Reply:
Not easily as they use different arguments to `Load`.